benchmark/benchmark_util/data_table_benchmark_util.cpp

```diff
@@ -77,7 +77,7 @@ LargeDataTableBenchmarkObject::LargeDataTableBenchmarkObject(const std::vector<u
       operation_ratio_(std::move(operation_ratio)),
       generator_(generator),
       layout_({attr_sizes}),
-      table_(common::ManagedPointer(block_store), layout_, storage::layout_version_t(0)),
+      table_(common::ManagedPointer<storage::BlockStore>(block_store), layout_, storage::layout_version_t(0)),
```

Comment:
i've had issues compiling on linux without those in the past? idk its really necessary but it used to fail CI back before this  was closed. I can try it without it if you want?

Reply:
It should only fail if there's a single argument to the constructor (in which case you need an explicit cast because single argument constructors must be declared `explicit` in our codebase).

Reply:
done